src/dinitctl.cc

```diff
@@ -760,27 +760,106 @@ static void print_termination_details(int exit_status)
     }
 }
 
-// Wait for a service to reached stopped (do_stop == true) or started (do_stop == false) state.
-// Returns 0 if the service started/stopped, 1 if start/stop was cancelled or failed.
-static int wait_service_state(int socknum, cpbuffer_t &rbuffer, handle_t handle,
+// Process a SERVICEEVENT packet if it is related to the specified service handle, and optionally report
+// the service status to the user (verbose == true).
+// The caller must ensure that a complete packet of type SERVICEEVENT is present in the buffer before
+// calling.
+// The size of current SERVICEEVENT should be provided as "pktlen".
```

Comment:
Here you have quotes around the parameter name (`"pktlen"`) but on the next line there are no quotes around another parameter name (`do_stop`). Please make sure changes are consistent - with existing practice, and with themselves.